query/src/lib.rs

```diff
@@ -499,12 +499,26 @@ pub enum PullConcreteAttribute {
     Entid(i64),
 }
 
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct NamedPullAttribute {
+    pub attribute: PullConcreteAttribute,
+    pub alias: Option<Rc<NamespacedKeyword>>,
+}
+
+impl From<PullConcreteAttribute> for NamedPullAttribute {
+    fn from(a: PullConcreteAttribute) -> Self {
+        NamedPullAttribute {
+            attribute: a,
+            alias: None,
+        }
+    }
+}
+
 #[derive(Clone, Debug, Eq, PartialEq)]
 pub enum PullAttributeSpec {
     Wildcard,
-    Attribute(PullConcreteAttribute),
+    Attribute(NamedPullAttribute),
     // PullMapSpec(Vec<…>),
-    // AttributeWithOpts(PullConcreteAttribute, …),
     // LimitedAttribute(PullConcreteAttribute, u64),  // Limit nil => Attribute instead.
```

Comment:
`PullConcreteAttribute` has been superceded, so update these comments to `NamedPullAttribute`.